docs/software/apple/translate.mdx

```diff
@@ -0,0 +1,18 @@
+---
+id: apple-app-translate
+title: Translate the Apple App
+sidebar_label: Translate
+sidebar_position: 3
+---
+
+## How to Contribute
+
+Contributing translations to the Meshtastic Apple app helps make the project accessible to a wider audience. Follow these steps to add translations for a new language:
+
+1. **Fork the Repository**: Start by forking the [Meshtastic-Apple repository](<https://github.com/meshtastic/Meshtastic-Apple/tree/main>) to your GitHub account.
+2. **Create a Language Folder**: In your forked repository, create a new folder for your language using the language code followed by `.lproj`. For example, for German, create a folder named `de.lproj`.
+3. **Copy the Localizable.strings File**: Navigate to the English strings folder [here](<https://github.com/meshtastic/Meshtastic-Apple/blob/main/en.lproj/Localizable.strings>) and copy the `Localizable.strings` file. Paste this file into the folder you created in the previous step.
```

Comment:
I'm personally a fan of linking the text that is meaningful vs "here". In this case that would mean just linking `English strings folder` and omitting the word `here` altogether 🍻